.github/workflows/qe-tests.yml

```diff
@@ -109,6 +109,19 @@ jobs:
               -qsvvv \
               tests/dogtag/pytest-ansible/pytest/sanity/test_role_users.py
 
+      - name: Run SCEP test
+        run: |
+          pip3 install -r tests/dogtag/pytest-ansible/requirements.txt
+          pip3 install -e tests/dogtag/pytest-ansible
+          pytest-3 \
+              --ansible-host-pattern master \
+              --ansible-inventory inventory \
+              --ansible-module-path tests/dogtag/pytest-ansible/common-modules \
+              --junit-xml pki_cert_scep_1664435_junit.xml \
+              -qsvvv \
+              tests/dogtag/pytest-ansible/pytest/ca/scep/test_bug_1664435_pki_ca_scep_enrollment.py
```

Comment:
Is it possible to merge this into the previous step (lines 100-110) so we don't have to keep modifying the workflow whenever we add new tests?

Reply:
I added this as per my previous job (https://github.com/dogtagpki/pki/pull/3351/files)

Should I add jobs as below?

------
              -qsvvv \
              tests/dogtag/pytest-ansible/pytest/sanity/test_role_users.py
              tests/dogtag/pytest-ansible/pytest/ca/scep/test_bug_1664435_pki_ca_scep_enrollment.py
------

Reply:
I'm thinking the SCEP test should be added into `tests/dogtag/pytest-ansible/pytest/sanity/test_role_users.py` so that if we want to run sanity test manually we only need to run just that one file (assuming SCEP test is part of sanity test).

Reply:
Done.